pkgs/development/python-modules/qiskit-ignis/default.nix

```diff
@@ -0,0 +1,66 @@
+{ lib
+, pythonOlder
+, buildPythonPackage
+, fetchFromGitHub
+, numpy
+, qiskit-terra
+, scipy
+  # Check Inputs
+, stestr
+, qiskit-aer
+}:
+
+buildPythonPackage rec {
+  pname = "qiskit-ignis";
+  version = "0.2.0";
+
+  disabled = pythonOlder "3.5";
+
+  # Pypi's tarball doesn't contain tests
+  src = fetchFromGitHub {
+    owner = "Qiskit";
+    repo = pname;
+    rev = version;
+    sha256 = "08a60xk5dq5wmqc23r4hr2v2nsf9hs0ybz832vbnd6d80dl6izyc";
+  };
+
+  prePatch = ''
+    # fix 0.2.0 error where stestr conf pointed to wrong dir name
+    substituteInPlace .stestr.conf --replace "./tests" "./test"
+  '';
+
+  propagatedBuildInputs = [
+    numpy
+    qiskit-terra
+    scipy
+  ];
+
+  doCheck = true;
+  pythonImportsCheck = [ "qiskit.ignis" ];
+  checkInputs = [
+    stestr
+    qiskit-aer
+  ];
+  dontUseSetuptoolsCheck = true;
+  preCheck = ''
+    echo "Moving tests & config to OUT"
+    cp /build/source/.stestr.conf $out
+    cp -r $src/test $out
+  '';
+  checkPhase = ''
+    runHook preCheck
+
+    cd $out && eval "stestr run"
+
+    runHook postCheck
```

Comment:
could probably just be
```
  stestr run
```

Reply:
Is there a shortcut for getting back to the build source directory (if using fetchFromGitHub: /build/source, but if using fetchPypi: /build/$pname-$version). Most of my issues are because it feels inconsistent where the tests are run from (i.e. what ``pwd`` returns). Sometimes it's $out, sometimes it's /build/source.